src/sentry/api/endpoints/team_alerts_triggered.py

```diff
@@ -0,0 +1,51 @@
+from django.db.models import Count, Q
+from django.db.models.functions import TruncDay
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases.team import TeamEndpoint
+from sentry.api.utils import get_date_range_from_params
+from sentry.incidents.models import (
+    IncidentActivity,
+    IncidentActivityType,
+    IncidentProject,
+    IncidentStatus,
+)
+from sentry.models import Project
+
+
+class TeamAlertsTriggeredEndpoint(TeamEndpoint, EnvironmentMixin):
+    def get(self, request, team):
+        """
+        Return a time-bucketed (by day) count of triggered alerts owned by a given team.
+        """
+        project_list = Project.objects.get_for_team_ids([team.id])
+        owner_ids = [team.actor_id] + [om.user.actor_id for om in team.member_set]
+        start, end = get_date_range_from_params(request.GET)
+        bucketed_alert_counts = (
+            IncidentActivity.objects.filter(
+                (
+                    Q(type=IncidentActivityType.CREATED.value)
+                    | Q(
+                        type=IncidentActivityType.STATUS_CHANGE.value,
+                        value__in=[
+                            None,
+                            IncidentStatus.OPEN,
+                            IncidentStatus.CRITICAL,
+                            IncidentStatus.WARNING,
+                        ],
+                    )
+                ),
+                incident__alert_rule__owner__in=owner_ids,
```

Comment:
Pretty happy with how the actor stuff simplifies queries like this

Reply:
Yeah, me too :)